lib/pennyworth/spec.rb

```diff
@@ -21,23 +21,23 @@
 require "open-uri"
 require "yaml"
 
-require_relative "exceptions"
-require_relative "helper"
-require_relative "pennyworth_libvirt"
-require_relative "runner"
-require_relative "vagrant"
-require_relative "vagrant_runner"
-require_relative "image_runner"
-require_relative "host_config"
-require_relative "host_runner"
-require_relative "lock_service"
-require_relative "ssh_keys_importer"
-require_relative "vm"
-require_relative "spec_profiler"
-require_relative "remote_command_runner"
-require_relative "settings"
-require_relative "local_runner"
-require_relative "local_command_runner"
+require "pennyworth/exceptions"
```

Comment:
I'm sorry but I realized that we actually need the require_relative change here, not in `pennyworth.rb` (although it doesn't hurt there either, so I'd keep it).